src/components/About/styles.scss

```diff
@@ -0,0 +1,148 @@
+// Mobile portrait
+.wrapper {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	background-color: blue;
+}
+
+	.hero {
+		display: block;
+		width: 100%;
+		height: 38%;
+	}
+
+	.textContainer {
+		width: 100%;
+		height: 62%;
+		--padding__body: 10vw;
+		padding: var(--padding__body) var(--padding__body) 0 var(--padding__body);
+		overflow-y: auto;
+		font-size: var(--font_size__body);
+		background-color: pink;
+	}
+
+		.heading, .subheading {
+			font-family: var(--font_family__heading);
+		}
+
+		.heading {
+			margin-bottom: 0.5rem;
+			font-size: var(--font_size__heading);
+		}
+
+		.subheading {
+			width: 85%;
+			margin-bottom: 1rem;
+			font-size: var(--font_size__subheading);
+			line-height: 1.5em;
+		}
+		
+		.body {
+			margin-bottom: var(--padding__body);
+			font-family: var(--font_family__body);
+			line-height: 1.5em;
+			background-color: green;
+
+			p + p {
+				margin-top: 1em;
+			}
+		}
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper {
+		flex-direction: row;
+	}
+
+		.hero {
+			width: 50%;
+			height: 100%;
+		}
+
+		.textContainer {
+			width: 50%;
+			height: 100%;
+			--padding__body: 5vw;
+		}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+	.hero {
+		height: 60%;
+	}
+
+	.textContainer {
+		height: 40%;
+		--font_size__factor: 1.62;
+		font-size: calc(var(--font_size__body)*var(--font_size__factor));
+	}
+
+		.heading {
+			font-size: calc(var(--font_size__heading)*var(--font_size__factor));
+		}
+
+		.subheading {
+			font-size: calc(var(--font_size__subheading)*var(--font_size__factor));
+		}
+
+		.body {
+			p {
+				width: 80%;
+			}
+
+			p:nth-child(even) {
+				margin-left: 4em;
+			}
+		}
+}
+
+// Tablet landscape
+@media (min-width: 768px) and (orientation: landscape) {
+	.hero {
+		height: 100%;
+	}
+
+	.textContainer {
+		height: 100%;
+		--font_size__factor: 1.25;
```

Comment:
aah, so that's why you did it - yeah, we can deffo use `em` cleverly here instead, and just be changing the font-size...might mean another wrapping `<div>` or something though. It's a clever use of the variables, but I think as we have `em` as a native way to scale fonts it feels like we should be using that instead. Grab me after work tomorrow and we'll find a way to do this.